tests/cbmc/stubs/cbmc_allocate_s2n_socket_write_io_context.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <utils/s2n_socket.h>
+
+struct s2n_socket_write_io_context *cbmc_allocate_s2n_socket_write_io_context()
+{
+    struct s2n_socket_write_io_context *s2n_socket_write_io_context = malloc(sizeof(*s2n_socket_write_io_context));
+    return s2n_socket_write_io_context;
+}
```

Comment:
(Same for other allocators) This function should go to https://github.com/aws/s2n-tls/blob/main/tests/cbmc/sources/make_common_datastructures.c